netlogo-gui/src/main/theme/InterfaceColors.scala

```diff
@@ -31,6 +32,7 @@ object InterfaceColors {
   private val MEDIUM_GRAY = new Color(175, 175, 175)
   private val LIGHT_GRAY_OUTLINE = new Color(120, 120, 120)
   private val DARK_GRAY = new Color(79, 79, 79)
+  private val DARK_GRAY_2 = new Color(100, 100, 100)
```

Comment:
I was using the `_2` suffix to indicate a slightly darkened variety of a color, so maybe for consistency this should be renamed.

Reply:
Deferring to the larger color theme reorganization